reconcile/utils/terrascript_client.py

```diff
@@ -933,21 +933,26 @@ def populate_tf_resource_rds(self, resource, namespace_info,
             }
             role_tf_resource = aws_iam_role(em_identifier, **em_values)
             tf_resources.append(role_tf_resource)
+            values['monitoring_role_arn'] = \
+                "${" + role_tf_resource.arn + "}"
```

Comment:
Since you are here, can you change this into `f'` strings?

Reply:
Done.
https://github.com/app-sre/qontract-reconcile/blob/98ef424662e180f647c1bcae93db8178184d600b/reconcile/utils/terrascript_client.py#L946